src/ts/address-search.ts

```diff
@@ -305,8 +305,8 @@ export default class AddressValidation {
     }
     return JSON.stringify(data);
   }
-
-  private generateLookupDataForApiCall(input: string, shouldGetSuggestions: boolean): string {
+  
+  private generateLookupDataForApiCall(input: string, lookupKeyword: string): string {

```

Comment:
Rather than the lookupKeyword parameter being of type string, it could be `AddressValidationLookupKeywords`.

Reply:
done